rbc/src/main/java/com/library/rbc/model/Book.java

```diff
@@ -0,0 +1,32 @@
+package com.library.rbc.model;
+
+import com.library.rbc.model.enums.BookCategory;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+
+@Document(collection = "books")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Book {
+
+    @Id
+    private String id;
+    private String isbn;
+    private String title;
+    private String description;
+    private Author author;
+    private String imageUrl;
+    private Integer totalNumberOfCopies;
+    private List<String> favourites; //collection of user ids
```

Comment:
Consider renaming the fields so that they are more clear. This would also reduce the need for the comments. 
Suggestion: `usersWhoFavorited` instead of `favourites`, `usersOnWaitingList` instead of `waitingList`, `usersWhoSuggested` instead of `suggestions`